types/graphql-upload/Upload.d.ts

```diff
@@ -0,0 +1,8 @@
+import { FileUpload } from './';
+
+export { FileUpload };
```

Comment:
This should not be reexported as the upload.mjs doesnt have any other reexports

Reply:
I have now duplicated the types that were previously in `index.d.ts` into each of the files that need them.
I'm not really happy with this, but I see no way to have common types in a separate file and at the same time following the structure of the original package.
Each file exports the types it defines as they are useful when consuming the library, for example to annotate variable and function types.